script.module.archive_tool/addon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.archive_tool" name="archive_tool" provider-name="Zach Morris" version="1.13">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.kodi-six"/>
+    <import addon="vfs.libarchive" version="1.0.5"/>
+    <import addon="vfs.rar" version="2.3.0" optional="true"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Python class for working with file archives</summary>
+    <description lang="en">archive_tool is a set of common tools to work with the Kodi libarchive virtual file system</description>
+    <platform>all</platform>
+    <assets>
+      <icon>icon.png</icon>
+    </assets>
+    <license>free</license>
```

Comment:
Such license doesn't exist. According to the License of your addon this should be `GPL-3.0-only` (spdx identifier)